crypto/dsa/dsa_lib.c

```diff
@@ -173,18 +173,20 @@ static DSA *dsa_new_intern(ENGINE *engine, OSSL_LIB_CTX *libctx)
 
 DSA *DSA_new_method(ENGINE *engine)
 {
-    return dsa_new_intern(engine, NULL);
+    /* make the compiler happy */
+    (void)engine;
```

Comment:
We usually put `ossl_unused` in the function prototype:
```c
DSA *DSA_new_method(ossl_unused ENGINE *engine)
```

Reply:
thanks, fixed